src/openms/include/OpenMS/CHEMISTRY/ElementDB.h

```diff
@@ -39,10 +39,10 @@
 #include <OpenMS/DATASTRUCTURES/String.h>
 #include <OpenMS/DATASTRUCTURES/Map.h>
 #include <OpenMS/CHEMISTRY/IsotopeDistribution.h>
+#include <OpenMS/CHEMISTRY/Element.h>
 
 namespace OpenMS
 {
-  class Element;
```

Comment:
any reason to do this? can you revert that and remove `#include <OpenMS/CHEMISTRY/Element.h>` again?